lib/sensors/wheel_encoder.hpp

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include <cstdint>
+#include <memory>
+#include <optional>
+
+#include <core/logger.hpp>
+#include <io/i2c.hpp>
+
+namespace hyped::sensors {
+
+// ! these values still have to be found
+constexpr std::uint8_t kDefaultWheelEncoderAddress = 0x0;
+
+constexpr std::uint8_t low_byte_address_    = 0x0;
+constexpr std::uint8_t middle_byte_address_ = 0x0;
+constexpr std::uint8_t high_byte_address_   = 0x0;
+
+constexpr std::uint8_t kFreeRegisterAddress = 0x0;
+constexpr std::uint8_t kFreeRegisterValue   = 0x2;

```

Comment:
these addressed need to be private `static constexpr` in the wheelencoder class itself since these never need to be accessed outside this class